plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -192,6 +192,12 @@ public String getFileName() {
   private JarArchiver jarArchiver;
 
   /**
+   * Allows enabling/disabling gcr credentials
+   */
+  @Parameter(defaultValue = "true", property = "dockerfile.googleregistry")
```

Comment:
Looking at this again, I was mistakenly thinking that `property` controlled the name of the `<configuration>` element in pom.xml. For a system property (what `property` actually does, IIRC) then I think the `dockerfile` prefix is ok, although the should maybe be `dockerfile. googleContainerRegistryEnabled`